castle/cms/_scripts/create-site.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import json
+import os
+
+import transaction
+from AccessControl.SecurityManagement import newSecurityManager
+from castle.cms.cron.utils import setup_site
+
+
+try:
+    from castle.cms.browser.site.addsite import AddCastleSite
+except ImportError:
+    # required! This could be getting called from upgrade
+    # integration tests
+    from castle.cms.browser.addsite import AddCastleSite
+
+
+app = app  # noqa
+
+parser = argparse.ArgumentParser(
+    description='...')
+parser.add_argument('--site-id', dest='site_id', default='Castle')
+parser.add_argument('--delete', dest='delete', default=False)
+args, _ = parser.parse_known_args()
+
+user = app.acl_users.getUser('admin')
+newSecurityManager(None, user.__of__(app.acl_users))
+
+if args.delete and args.site_id in app.objectIds():
+    app.manage_delObjects([args.site_id])
```

Comment:
Feel like some feedback here would be useful....just in case

Reply:
done.